src/index.js

```diff
@@ -365,17 +365,33 @@ async function processWebhook({ msg, client }) {
             //sending files if there is any 
             if (itemResponse.files && itemResponse.files.length > 0) {
                 itemResponse.files.forEach(async (itemFile) => {
+
                     // #TODO: Passing same mimetype for all images
-                    var media = await new MessageMedia(
-                        "image/jpg",
+                    // var imageMedia = await new MessageMedia(
+                    //     "image/jpg",
+                    //     itemFile.file,
+                    //     itemFile.name
+                    // );
+
+                    // #TODO: Passing same mimetype for all pdf files
+                    var pdfMedia = await new MessageMedia(
+                        "application/pdf",
```

Comment:
can't keep `"application/pdf"` hard coded. All the images will also be sent as pdf extension which won't open on receiver's end. 

better to take it from base64 string like this 
`data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAgAAAAIACAYAAAD0eNT6AAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAOwwAADs.....` split this string by `;` and you will get your mime type. 

if user doesn't provide any mimeType then take  `image/jpg` as default